src/app/DTT/address/get_address/helpers/set-address-suggestions.js

```diff
@@ -0,0 +1,8 @@
+import { equals } from 'ramda'
+const setAddressSuggestions = (searchAddress, allOptions, setAllOptions) => {
+  const displaySuggestions = predictions =>
```

Comment:
the name of this function is displaySuggestions but I don't see it rendering anything. Maybe we should call it setAllOptionsFromPredictions